src/Orchard.Cms.Web/Startup.cs

```diff
@@ -52,6 +54,7 @@ public void Configure(IApplicationBuilder app, IHostingEnvironment env, ILoggerF
             app.UseStaticFiles();
 
             loggerFactory.AddConsole(Configuration);
+            loggerFactory.AddDebug();
```

Comment:
It **should**. The console listener is useless when debugging an app hosted behind IIS (because well... there's no console).